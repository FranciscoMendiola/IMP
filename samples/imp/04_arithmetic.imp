resultado := a + b * c - d
