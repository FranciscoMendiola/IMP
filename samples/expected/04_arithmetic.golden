Token {tokenKind = Id, lexeme = "resultado", position = 0}
Token {tokenKind = Assign, lexeme = ":=", position = 10}
Token {tokenKind = Id, lexeme = "a", position = 13}
Token {tokenKind = Plus, lexeme = "+", position = 15}
Token {tokenKind = Id, lexeme = "b", position = 17}
Token {tokenKind = Times, lexeme = "*", position = 19}
Token {tokenKind = Id, lexeme = "c", position = 21}
Token {tokenKind = Minus, lexeme = "-", position = 23}
Token {tokenKind = Id, lexeme = "d", position = 25}
